L5

bitoffluff1 opened the pull request.

bitoffluff1 committed:
Дз 5
Форма регистрации новых пользователей
Дата окончания события если пуста, то равна дате начала
И проверка даты окончания что она позже создания

bitoffluff1 committed:
Редактирование события

manchenkoff commented on controllers/ActivityController.php:
```diff
@@ -34,34 +50,31 @@ public function actionView($id = 1)
-    public function actionSubmit()
+    public function actionSubmit($id = '')
     {
         $model = new Activity();
```
вот тут если передан $id - нужно подгружать существующую модель для обновления

bitoffluff1 committed:
Исправила метод submit(activity)
Переписала метод create -> edit(activity)
Скорректировала метод signup(site)
Сделала несколько изменений в БД с помощью миграций:
 - изменила название таблицы Activities на Activity
 - изменила все названия столбцов из кемелкейс в снейкейс

Signed-off-by: bitoffluff1 <Lena_khmelnikova@mail.ru>
